### PR TITLE
Created rough profile header for use on guide and user profile pages.

### PR DESCRIPTION
**Pull request by RyuGuardian**

### CODE REVIEW

File: app/components/ProfileHeader/index.js
Comment (Octember) [inline app/components/ProfileHeader/index.js:22]: h1 className h2? I don't think that h2 classname does anything

File: app/components/ProfileHeader/index.js
Comment (Octember) [inline app/components/ProfileHeader/index.js:26]: h5 classname? What's the point of that?

File: app/components/ProfileHeader/index.js
Comment (Octember) [inline app/components/ProfileHeader/index.js]: `<h3>` instead of classname h3, right?

File: app/components/ProfileHeader/index.js
Comment (Octember) [inline app/components/ProfileHeader/index.js]: What is `arr`? Looks unused, can we remove?

File: app/components/ProfileHeader/index.js
Comment (Octember) [inline app/components/ProfileHeader/index.js]: This is way too complicated to render inline. If we needed this functionality I'd want it to be broken out into a separate function

File: app/containers/GuideProfilePage/index.js
Comment (Octember) [inline app/containers/GuideProfilePage/index.js:61]: Can you put it on desktop too?

File: app/containers/GuideProfilePage/index.js
Comment (Octember) [inline app/containers/GuideProfilePage/index.js]: The API already exists for this - you don't need to do this now but we'll need to integrate that call into the props

File: app/components/ProfileHeader/index.js
Comment (Octember) [reply app/components/ProfileHeader/index.js]: Instead of this, let's make the `stats` object be of a shape, e.g.

```
stats: PropTypes.objectOf(PropTypes.shape({
    count: PropTypes.number,
    label: PropTypes.string,
  })
```

In general, I don't want any complicated logic on the frontend. It will sometimes be needed, but encoding business logic here is a recipe for bugs down the line IMO

**Review by Octember — CHANGES_REQUESTED** [review]
The page looks great. Some philosophical requests. I'll make a ticket to plumb the data into this.

File: app/components/ProfileHeader/index.js
Comment (RyuGuardian) [reply app/components/ProfileHeader/index.js:22]: It styles it like an h2 (Bootstrap's h2). It's a simple way to style text to a certain size and give it margins without messing with the HTML semantics or making a new styled component.

File: app/components/ProfileHeader/index.js
Comment (RyuGuardian) [reply app/components/ProfileHeader/index.js]: Oops, one of those things I thought I was gonna use then didn't.

File: app/components/ProfileHeader/index.js
Comment (RyuGuardian) [reply app/components/ProfileHeader/index.js]: OK. Wasn't sure what would be best to do here, so just went with my first thought and finished it to get it done and turn in (I also like working with RegEx).

File: app/containers/GuideProfilePage/index.js
Comment (RyuGuardian) [reply app/containers/GuideProfilePage/index.js]: Yeah, I did this right before you announced the API. I'll integrate it with the Guide Page before moving on to something else.

File: app/components/ProfileHeader/index.js
Comment (RyuGuardian) [reply app/components/ProfileHeader/index.js]: I've looked up many times how h1-h6 should be used, and there doesn't seem to be a definitive guide, but I do think a heading should be a title rather than data like a number. Since the numbers here are large, they need the styling of a heading, though, and that's all the class is for.

File: app/components/ProfileHeader/index.js
Comment (RyuGuardian) [reply app/components/ProfileHeader/index.js:26]: Font sizing (see previous comment). My logic here is that the heading is significant enough to be an h2 (rather important), but since the mock-up showed a smaller font, it was simpler to use Bootstrap's h5 class (as recommended) rather than make a new styled component.

File: app/containers/GuideProfilePage/index.js
Comment (RyuGuardian) [reply app/containers/GuideProfilePage/index.js:61]: NP; no styling for desktop is set up, though.

File: app/containers/GuideProfilePage/index.js
Comment (Octember) [reply app/containers/GuideProfilePage/index.js]: I made a separate ticket for it, feel free to do that as a separate PR

File: app/components/ProfileHeader/index.js
Comment (Octember) [reply app/components/ProfileHeader/index.js:22]: Sorry I'm still confused, what do you mean "messing with HTML semantics"? Do you mean for like SEO, crawlers?

Comment (RyuGuardian) [reply app/components/ProfileHeader/index.js:22]: I don't have much knowledge on SEO and crawlers, but yeah, I think this is along the lines of the principles "HTML elements matter" (when considering semantic HTML) and "class names don't matter" (which seems to be the Bootstrap approach to styling?).

I'm trying to use the advantages Bootstrap gives me and give thought into the types of elements used in a page (I think it's also important for screen readers).

Comment (Octember) [reply app/components/ProfileHeader/index.js:22]: Interesting. Good thinking, I think you're probably right! Big props

Comment (RyuGuardian) [reply app/components/ProfileHeader/index.js:22]: Thanks!

**Comment by RyuGuardian** [comment]
Things should be good to go now after review and merging. Will work on integrating Guide API now.

File: app/containers/GuideProfilePage/index.js
Comment (Octember) [inline app/containers/GuideProfilePage/index.js:29]: nice

**Review by Octember — APPROVED** [review]

**Comment by Octember** [comment]
Looks good! Gonna merge this so you have a clean state for the next piece

nice work dude

**Pull request merged**
